articles/142_idl.md

```diff
@@ -35,3 +35,62 @@ Further it describes how the interfaces are used to generate code in C, C++ and
 <div class="alert alert-warning" markdown="1">
   <b>TODO:</b> All of this too
 </div>
+
+### Constraint Checking
+
+Generated code for messages is guaranteed to enforce constraints only when a message is published.
+For example, there might not be an error when a field constrained to be a fixed size array is assigned an array with the wrong size.
+In the future constraints might be checked when a field is set.
```

Comment:
Maybe append the following?

> ... and/or mutated.

Same for the next line.

Reply:
9a56e5d78d69a52fa04f159b7e9683030b1896f0 and 46ec1098fbc09fb93278003db8185ca45d37a72b